dagshub/data_engine/model/metadata_field_builder.py

```diff
@@ -80,6 +80,57 @@ def set_annotation(self, is_annotation: bool = True) -> "MetadataFieldBuilder":
         self._set_or_unset(ReservedTags.ANNOTATION.value, is_annotation)
         return self
 
+    def set_thumbnail(self, thumbnail_type: Union[ThumbnailType, None] = None,
+                      is_thumbnail: bool = True) -> "MetadataFieldBuilder":
+        """
+        Mark or unmark the field as thumbnail field, with the specified thumbnail type
+        """
+        if not is_thumbnail:
+            self._set_or_unset(ReservedTags.THUMBNAIL_VIZ.value, False)
+            return self
+
+        if thumbnail_type is None:
+            raise ValueError("Thumbnail type must be specified")
+
+        tag: ReservedTags
+
+        if thumbnail_type == ThumbnailType.VIDEO:
+            tag = ReservedTags.VIDEO_THUMBNAIL_VIZ
+        elif thumbnail_type == ThumbnailType.AUDIO:
+            tag = ReservedTags.AUDIO_THUMBNAIL_VIZ
+        elif thumbnail_type == ThumbnailType.IMAGE:
+            tag = ReservedTags.IMAGE_THUMBNAIL_VIZ
+        elif thumbnail_type == ThumbnailType.PDF:
+            tag = ReservedTags.PDF_THUMBNAIL_VIZ
+        elif thumbnail_type == ThumbnailType.TEXT:
+            tag = ReservedTags.TEXT_THUMBNAIL_VIZ
+        elif thumbnail_type == ThumbnailType.CSV:
+            tag = ReservedTags.CSV_THUMBNAIL_VIZ
+        else:
+            raise ValueError("Invalid thumbnail type")
+
+        self._set_or_unset_thumbnails(tag, True)
+        return self
+
+    def _set_or_unset_thumbnails(self, type_tag, is_thumbnail):
```

Comment:
The `is_thumbnail` arg in this function is only set to True, so the code in 131 will never run. 

I think you should remove the arg and inline the function back into `set_thumbnail`

Reply:
No its not true, in line 89 I call it with false
<img width="706" alt="image" src="https://github.com/DagsHub/client/assets/107917894/9b1f76ec-2b81-492e-893b-69594a118c20">